### PR TITLE
Debugging file

### PR DESCRIPTION
**Pull request by mohamed401**
add code foe debugging

### CODE REVIEW

File: tes67.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+print(+----+)
```
Comment (1kalisem1) [inline tes67.py]: You almost had it! You get an syntax error because you left out the parentheses for "+----+" in line 1. Fix that and you should be good!

**Comment by 1kalisem1** [comment]
It should work now.

**Comment by mohamed401** [comment]
okey and accepting and closing

**Comment by mohamed401** [comment]
okey and accepting and closing

**Pull request closed without merging**